pages/en/about-mina/faq.mdx

```diff
@@ -0,0 +1,28 @@
+import Page from "@reason/pages/Docs";
+import DocLink from "@reason/components/DocLink";
+export default Page({ title: "FAQ" });
+
+# FAQ
+Read answers to popular questions about Mina Protocol
+
+### What language is Mina written in?
```

Comment:
"The Mina node is written in OCaml. But you don't need to know OCaml to write smart contracts for Mina."